### DISCUSSION TITLE
docker memory

### DISCUSSION THREAD

**Discussion by wangqingpen** [edgexfoundry/community#241] [Q&A]
I use go to call c to get data, I can run for a long time when I use binaries, but there will always be memory problems when I use docker, for example：
fatal: morestack on g0
SIGSEGV: segmentation violation
PC=0x472894 m=8 sigcode=1.
 Your framework doesn't have a memory limit on docker

**Comment by cloudxxx8** [comment]
are you able to confirm which container consuming all the memory?
`docker stats` may be helpful.
